UT4MasterServer.Web/src/pages/Admin/Clients/components/AddClient.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <LoadingPanel :status="status" :error="errorMessage">
+        <form :class="{ 'was-validated': submitAttempted }" @submit.prevent="handleSubmit" novalidate>
+            <fieldset>
+                <legend>Add Client</legend>
+                <div class="form-group row">
+                    <label for="name" class="col-sm-12 col-form-label">Name</label>
+                    <div class="col-sm-6">
+                        <input type="text" class="form-control" id="name" name="name" v-model="name" v-valid="formValid" required />
+                        <div class="invalid-feedback">A unique name is required</div>

```

Comment:
Name is not really required for backed to work. It is meant as just some description for visual display.

Reply:
Yeah I know it isn't required in the backend, but without it you would have a dropdown with 32 digit IDs in it, not the easiest to use. I made it required because it would be a horrible pain to use if you didn't set a name.